src/config.rs

```diff
@@ -2885,11 +2900,11 @@ mod tests {
         );
     }
 
-    fn new_engines(cfg: TiKvConfig) -> (RocksEngine, ConfigController) {
+    fn new_engines(cfg: TiKvConfig, db_opt: DBOptions) -> (RocksEngine, ConfigController) {
         let engine = RocksEngine::from_db(Arc::new(
             new_engine_opt(
                 &cfg.storage.data_dir,
-                DBOptions::new(),
+                db_opt,
```

Comment:
the `db_opt` should be created from `TiKvConfig` instead of passing from parameter?